dev/tests/functional/tests/app/Magento/Newsletter/Test/Handler/Template/Curl.php

```diff
@@ -8,7 +8,6 @@
 
 use Magento\Mtf\Fixture\FixtureInterface;
 use Magento\Mtf\Handler\Curl as AbstractCurl;
-use Magento\Mtf\Util\Protocol\CurlTransport;
```

Comment:
CurlTransport is used in this class

Reply:
Please see answer below ... this class does not exist.